megamek/src/megamek/client/ui/swing/unitDisplay/WeaponPanel.java

```diff
@@ -2053,8 +2053,8 @@ else if (atype.getMunitionType() == AmmoType.M_IATM_IMP)
         }
 
         // Aero
-        if (entity.isAirborne() 
-                || entity.usesWeaponBays()) {
+        if (entity.isAirborne()){
+                //|| entity.usesWeaponBays()) {
```

Comment:
should this be removed?

Reply:
What happens when you have a grounded dropper that still uses weapon bays?  I think that's what the || statement was there for.

Reply:
Do we know where the rules for grounded dropships weapon ranges actually are?

Reply:
Errata actually changes them.  It's p249 of TW:

Attacks by DropShips are carried out with the exceptions noted above concerning firing arcs for grounded spheroid DropShips. Resolve each individual weapon as a separate attack, using ranges and damage values as found on pp. 303-304.

So no more bay fire by grounded droppers.

Reply:
Fun.  Guess I know what my next task is...   At least we have the framework for it already in place - the trick is how to handle bay ammo so everything is linked properly once the dropper lifts off

Reply:
On the plus side, the ranges from grounded dropship weapon bays never displayed correctly anyway, from what I can tell.